runtime/js/40_testing.js

```diff
@@ -198,15 +198,16 @@
           }
           ArrayPrototypePush(details, message);
         } else if (dispatchedDiff < completedDiff) {
-          const [name] = OP_DETAILS[key];
+          const [name, hint] = OP_DETAILS[key] || [key, null];
```

Comment:
This exposes the name of the op like the below error message:

```
- 1 async operation to op_write was started before this test, but was completed during the test. Async operations should not complete in a test if they were not started in that test.
```

I think the proper fix here is to add the entry for `op_write` in `OP_DETAILS` table.

Reply:
This is actually fine, I thought I fixed it in https://github.com/denoland/deno/pull/14074 but I missed this

Reply:
You are right though that we should fill in details for not listed ops.